### DISCUSSION TITLE
Lumbarian Empire Storyline

### DISCUSSION THREAD

**Discussion by OcelotWalrus** [OcelotWalrus/Cromha-Expansion-plugin#117] [General]
#### It's basically where you can give ideas, propose concepts and just talk about the storyline mission of the [Lumbarian Empire](https://github.com/OcelotWalrus/Cromha-Expansion-plugin/discussions/110)!

A potential storyline is written below, by @lumbar527. Suggestions are still welcome, but it would be greatly appreciated if you helped write the code.

<details><summary>Outdated Potential Storyline</summary>
<p>

# Lumbarian Storyline
Intro:
- Help the Lumbarian Empire convince "Republic" planets to join them. (After there is no more Republic)
- **Receive: Lumbarian License**
- Deep and Free Worlds refuse to join. The Lumbarian Empire ignores the Syndicate.
- Invade the Free Worlds because of a small FW attack.
- Invade the Deep to take technologies.
- **Receive: Lumbarian Knight License**

Start:
- Travel around to help the Lumbarian Empire make super-powerful laser batteries.
- Search the Deep for secret technology stashes.
- Work on reverse-engineering Pug cloaking tech from a secret Deep stash.
- Sneak to and from Empire Rebels space trading tech. (Things like Solar Cores, Lumbarian Cloaking Systems [Not coded], Lumba Repair Devices, Lumba Heavy Lasers, and Lumbarian turret batteries. Probably some more Rebels stuff but I don't know a ton about them.)
- Lead an Empire Rebels fleet around searching for Empire Navy patrol fleets. Destroy the ones you find. [Lasts a while]
- **Reward: A Lumbarian (heavy) cruiser, no license to buy it though.**

Middle:
- You're fighting the Cromha now:
- Lead attack fleets into Cromha space.
- Defend Lumbarian space/human space.
- Begin open trade with the Syndicate, selling them powerful designs which they copy and receiving more Korath supplies - Jump Drives, reactors, Met Par Tek 53s, and other useful things.
- Assist the Syndicate defeat a great invasion.
- **Reward: Lumbarian King License**
- Participate in a massive attack to take some Cromha planets. Steal their technologies.
- Travel to the Rebels and Syndicate to share Cromha tech with them.
- It's time to make the Lumbarian Empire truly great: Invade Hai space with a large attack fleet. They're already mad at you because of the Great Alliance, so it's probably fine.
- Wait a minute - what about the Quarg? Quickly travel "South" to defeat a Quarg attack, then push further. (You stop before the ringworld, because many Quarg there.)
- Travel...further "South".
- Meet these Heliarch and Coalition. Gain their alliance - the Lumbarian-Heliarch Union.
- Take a massive Heliarch invasion fleet to the Quarg ringworld. Destroy the Quarg there.
- Travel around human space gathering up the greatest fleet ever. The fleet uses Lumbarian ships, Heliarch ships (which now includes captured Quarg ships), powerful Syndicate ships (that they build with Korath/Cromha/Lumbarian stuff), and many regular military vessels (FW and Republic).
- Lead the fleet to Empire Space.
- WIN! (Maybe the Cromha Empire retreats to the Sentar Galaxy/Sherani space with the Far Remnant.)
- **Reward: Lumbarian Emperor License**

End:
- Establish a new peace treaty with the Hai, Wanderers, and Quarg.
- Work on making friendship with them.
- Finally make the Greater Alliance of the Lumbarian Empire, the Coalition, the Hai, the Wanderers, and the Quarg.

</p>
</details>

**Comment by lumbar527** [comment]
About the Heliarch/Coalition thing: is the treaty with the Great Alliance going to happen before or after the Lumbarian Empire discovers them? I think it should be, actually. And then I'll just replace mentions of it in the storyline listed here with the G[][]'[]a[]. (Nope, no more letters)

**Reply by OcelotWalrus** [reply]
I don't think the Lumbarian Empire will ever found about the Coalition, because the Cromha Empire and the Quarg will make sure they don't discover the Coalition Space.

**Reply by lumbar527** [reply]
OK, so it'll just be the G[][]'[]a[]. I've just started working on a ship now.

**Reply by OcelotWalrus** [reply]
Nice! Tell me when it's done!

**Comment by OcelotWalrus** [comment]
By the way, thanks you continuing the wiki, I saw it!

**Reply by lumbar527** [reply]
Not done yet, but working on it. Thanks!

**Reply by OcelotWalrus** [reply]
I've done some work on the Cromha outfits wiki page too. But I'm now coding the plugin, that's funnier :)

**Reply by lumbar527** [reply]
I don't understand.

**Reply by OcelotWalrus** [reply]
Sorry I written I'm done instead of I've done

**Comment by lumbar527** [comment]
Do you think it's OK that I'm filling Lumbarian stuff with direct Tolkien references? Does trademarking names apply to Endless Sky plugins?

**Reply by OcelotWalrus** [reply]
I think that's okay. You could say that the Lumbarian Emperor is a descendant or J.R.R Tolkien and the Emperor wishes to keep his ancestor work in the human space.

And I actually don't think that The Lord Of The Rings (the book) is copyrighted in any way.

**Reply by lumbar527** [reply]
> And I actually don't think that The Lord Of The Rings (the book) is copyrighted in any way.

OK, so that could be fine. I don't know about Ecthelion though, because I think he's in the Silmarillion, or at least one of the other books. (_Fall of Gondolin_)

**Reply by lumbar527** [reply]
Maybe Emperor Lumbar is a descendant of elves.

**Reply by OcelotWalrus** [reply]
Sure...
